tests/C/main.cpp

```diff
@@ -1,5 +1,60 @@
 #include <gtest/gtest.h>
```

Comment:
Let's follow [this style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes).